### DISCUSSION TITLE
Clon de Django

### DISCUSSION THREAD

**Discussion by GustavoGirardelli** [community/community#78138] [Other Feature Feedback, Questions, & Ideas]
Hola:
La idea es trabajar en Django en entornos virtuales.

$ git clone https://github.com/YourGitHubName/django.git
Clonando en 'django'...
Username for 'https://github.com': GustavoGirardelli
Password for 'https://GustavoGirardelli@github.com': 
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Autenticación falló para 'https://github.com/YourGitHubName/django.git/'

Pero no me deja descargar el clon de Django
Gracias

**Comment by EDMPhoenix** [comment]
La autenticación de contraseña para GitHub se eliminó el 13 de agosto de 2021. Ahora deberás utilizar un token de acceso personal o una llave SSH para clonar repositorios o interactuar con GitHub.

Para resolver este problema, puedes seguir estos pasos:

**Generar un nuevo token de acceso personal**

1. Ve a Github en tu navegador.
2. Haz clic en tu foto de perfil en la esquina superior derecha y selecciona Settings.
3. En el menú lateral izquierdo, selecciona Developer settings.
4. En el menú lateral izquierdo, selecciona Personal access tokens.
5. Haz clic en Generate new token.
6. Dale un nombre a tu token, establece el tiempo de expiración que prefieras y selecciona los permisos que necesitas. Para las operaciones básicas de Git, solo necesitas el ámbito de "repo" que incluye: repositorio público y privado, repositorio de la organización, repositorio de la colaboración, etc.
7. Haz clic en Generate token al final de la página.
8. Asegúrate de copiar tu nuevo token de acceso personal ahora. No podrás verlo de nuevo después de salir de la página.

**Usar el token de acceso personal en la línea de comandos**

Ahora, cuando vayas a clonar un repositorio, en lugar de usar tu contraseña, debes usar este token de acceso personal. Tu token de acceso personal ahora reemplaza tu contraseña.

```
$ git clone https://github.com/YourGitHubName/django.git
Username for 'https://github.com': YourGitHubName
Password for 'https://YourGitHubName@github.com': paste-your-personal-access-token-here
```

Nota: Asegúrate de reemplazar "paste-your-personal-access-token-here" con tu token de acceso personal. 

Eso debería resolver tu problema.

**Comment by ghostinhershell** [comment]
Hi @GustavoGirardelli, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
